### PR TITLE
New operation screen and create transaction

### LINKED ISSUE #27: Реализовать раскрытое состояние ячейки даты в NO
Необходимо отрисовать кастомный date picker

### LINKED ISSUE #29: Сделать добавление категории
Необходимо создать вью добавления категории 
И необходимо реализовать логику сохранения категории в кордату

### LINKED ISSUE #30: Реализовать раскрытое состояние ячейки тегов в NO

### PR DESCRIPTION
**Pull request by VladimirBrejcha**
Closes #27 
Closes #30 
Closes #29 

### Summary

- Added CreateTag, CreateCategory views and logic
- Added TagsContatiner, CategoryContainer views and logic
- Added NewOperation view and logic
- Disable Light mode in whole application

### Screenshots

image
image
image
image
image
image
image

### CODE REVIEW

**Review by antonhoang — APPROVED** [review]

File: bonsai/Views/FlexibleView/FlexibleView.swift [hunk 1]
```diff
@@ -0,0 +1,32 @@
+// Thx
+// https://github.com/FiveStarsBlog/CodeSamples/tree/48e493a2b4acd7196c176689a8f3038936f0ed41/Flexible-SwiftUI
+
+import SwiftUI
+
+/// Facade of our view, its main responsibility is to get the available width
+/// and pass it down to the real implementation, `_FlexibleView`.
```
Comment (Valter4578) [inline bonsai/Views/FlexibleView/FlexibleView.swift]: At the first glance I don't see any reasons to divide this view into two classes. I'll try to combine them myself and figure out why they're divided

**Comment by Valter4578** [comment]
Screenshot 2022-06-23 at 23 35 27
I don't like standard Colors for nav bar button in the screen where we create a new tag. It stands out a lot. Make them purple like on previous screen.

**Comment by VladimirBrejcha** [comment]
@Valter4578 Ебать где софты

**Comment by Valter4578** [comment]
Simulator Screen Shot - iPhone 13 - 2022-06-23 at 23 37 30
Incorrect placeholder behaviour. When text field selected there are both cursor and placeholder

**Comment by Valter4578** [comment]
Simulator Screen Shot - iPhone 13 - 2022-06-23 at 23 39 22
Same problem with new category screen. Standard nav bar buttons' colours.

**Comment by Valter4578** [comment]
Screenshot 2022-06-23 at 23 41 17

The whole button isn't selectable. Only area with arrow and text. Whitespace should be clickable too

**Comment by Valter4578** [comment]
In general, really good work. But there are a couple of small improvements needed.

**Review by Valter4578 — CHANGES_REQUESTED** [review]
In general, really good work. But there are a couple of small improvements needed.

**Comment by VladimirBrejcha** [comment]
> Simulator Screen Shot - iPhone 13 - 2022-06-23 at 23 37 30
> Incorrect placeholder behaviour. When text field selected there are both cursor and placeholder

Minor, not sure that fix needed. Lets leave it yet

**Review by Valter4578 — APPROVED** [review]

**Pull request merged**